mmdet/ops/merge_cells.py

```diff
@@ -0,0 +1,125 @@
+from abc import abstractmethod
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+
+class BaseMergeCell(nn.Module):
+
+    def __init__(self,
+                 channels=256,
+                 binary_op_type='sum',
+                 with_conv=True,
+                 conv_cfg=dict(),
+                 norm_cfg=None,
+                 order=('act', 'conv', 'norm'),
+                 with_input_conv_x=False,
+                 with_input_conv_y=False,
+                 input_conv_cfg=None,
+                 input_norm_cfg=None,
+                 resize_methods='interpolate'):
+        super(BaseMergeCell, self).__init__()
+        self.with_conv = with_conv
+        self.with_x_conv = with_input_conv_x
+        self.with_y_conv = with_input_conv_y
+        self.resize_methods = resize_methods
+
+        if self.with_conv:
+            groups = conv_cfg.pop('groups', 1)
+            kernel_size = conv_cfg.pop('kernel_size', 3)
+            padding = conv_cfg.pop('kernel_size', 1)
+            bias = conv_cfg.pop('bias', True)
```

Comment:
Setting many default values in the code is not the best way. Expose them in the arguments of the method.